.ruby-version

```diff
@@ -1 +1 @@
-2.6.6
+2.7.7
```

Comment:
Updating the default Ruby version to align with RuboCop target version.

https://github.com/HiMamaInc/payrix/blob/43d64a2771bb7b04f40e2145e433c7a4b6d29a84/.rubocop.yml#L6-L7